### DISCUSSION TITLE
Welcome to rest.nvim Discussions!

### DISCUSSION THREAD

**Discussion by NTBBloodbath** [rest-nvim/rest.nvim#27] [Announcements]
## 👋 Welcome aboard!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Feel free to ask anything related to rest.nvim and HTTP requests here even if they're very basic things, we are here to help you!

**Comment by pr4sh4nn4-7** [comment]
I install rest.nvim using :Rocks install but stilll nvim show :Rest command is unknown
